storage/aptosdb/src/schema/mod.rs

```diff
@@ -48,6 +49,7 @@ pub const TRANSACTION_BY_HASH_CF_NAME: ColumnFamilyName = "transaction_by_hash";
 pub const TRANSACTION_INFO_CF_NAME: ColumnFamilyName = "transaction_info";
 pub const VERSION_DATA_CF_NAME: ColumnFamilyName = "version_data";
 pub const WRITE_SET_CF_NAME: ColumnFamilyName = "write_set";
+pub const MIN_READABLE_VERSION_INDEX_CF_NAME: ColumnFamilyName = "min_readable_version_index";
```

Comment:
I'd make and name this table more general purpose. "db_status", "progress", "misc_records", etc. 
@grao1991 @lightmark : help on naming?

Reply:
db_metadata?

Reply:
Rename.